lib/config/configuration.go

```diff
@@ -1751,6 +1751,9 @@ kubernetes matchers are present`)
 				AssumeRole: assumeRole,
 			})
 		}
+		if fc.Discovery.AccessGraph.PollInterval > 0 {
+			tMatcher.PollInterval = types.Duration(fc.Discovery.AccessGraph.PollInterval)
```

Comment:
```suggestion
			tMatcher.PollInterval = fc.Discovery.AccessGraph.PollInterval
```

Reply:
Whoops, just realized this conversion is necessary between `time.Duration` and the protobuf's `types.Duration`.

Reply:
If you compile the code again it won't be needed